scripts/classes.py

```diff
@@ -0,0 +1,51 @@
+class Module:
```

Comment:
Add some documentation about the classes.

Reply:
https://github.com/erlerobot/HRIM/commit/dba716feba173fbdc2e0c0e6db1e783296aebed3

May be a bit excessive (I feel like many of them are self-explanatory), doesn't hurt to be too specific.